spec/lib/facebook_data_analyzer_spec.rb

```diff
@@ -0,0 +1,98 @@
+RSpec.describe FacebookDataAnalyzer do
+  before(:all) { described_class.run(catalog: test_catalog) }
+
+  describe  '.run' do
+    context '.xlsx' do
+      let!(:workbook) { Workbook::Book.open 'facebook_data_analyzer.xlsx' }
+
+      it 'generates "Friends ranking" worksheet correctly' do
+        sheet = workbook[0]
+        table = sheet.table
+        expect(table["B3"].value).to eq("Suzanne Nash")
```

Comment:
Use single quotes `'some text'`. Use double quotes when interpolating strings `"#{my_value} some text"` (rubocop would complain)

eg.
` expect(table["B3"].value).to eq("Suzanne Nash")` ->  `expect(table['B3'].value).to eq('Suzanne Nash')`